src/runtime/virtcontainers/pkg/types/pcipath_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2020 Red Hat
```

Comment:
> The form I have is what's recommended by Red Hat for open source contributions.

:confused: :interrobang:   ... 2021 ?

Reply:
Oops, I missed this one - I've updated it to match the others as simply "Copyright Red Hat.".